README.md

```diff
@@ -36,7 +36,7 @@ If you need support about something related to the website, plese join `#monero-
 Once you have the above list of things, it's typically a good idea to build the website from your local computer to make sure it works before you make any changes. To do this, complete the following steps:
 
 1. Navigate to your local `monero-site` repository.
-2. Serve the website: `bundle exec jekyll serve --baseurl ''`. If you want, you can speedup thi process by loading only the last blog post instead of all of them. Simply add `--limit_posts 1` to the command above. The resulting command will be `bundle exec jekyll serve --limit_posts 1 --baseurl ''`.
+2. Serve the website: `bundle exec jekyll serve --baseurl ''`. If you want, you can speedup the process by loading only the last blog post instead of all of them. Simply add `--limit_posts 1` to the command above. The resulting command will be `bundle exec jekyll serve --limit_posts 1 --baseurl ''`.
```

Comment:
The changes to the readme are not related to the commits. Please remove them or edit the commit description